Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+#!groovy
+
+@Library('pipeline-library') _
+
+def img
+
+node {
+    stage('build') {
+        checkout(scm)
+        img = buildApp(name: 'hypothesis/py_proxy')
+    }
+
+    stage('test') {
+        testApp(image: img, runArgs: '-u root') {
+            sh 'pip install -q tox>=3.8.0'
+            sh 'cd /var/lib/py_proxy && tox -e py36-tests'
+        }
+    }
```

Comment:
We'll need to run code formatting, lint and coverage on Jenkins as well. See LMS for an example: https://github.com/hypothesis/lms/blob/master/Jenkinsfile